src/components/Composer/index.tsx

```diff
@@ -73,6 +73,7 @@ function Composer(
         isComposerFullSize = false,
         shouldContainScroll = true,
         isGroupPolicyReport = false,
+        showSoftInputOnFocus = true,
```

Comment:
This is the change compared to the previous PR. In this PR, we add the default value is true for showSoftInputOnFocus